llama-index-integrations/llms/llama-index-llms-upstage/pyproject.toml

```diff
@@ -35,9 +35,9 @@ version = "0.2.1"
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 openai = "^1.21.2"
-llama-index-llms-openai = "^0.2.0"
 tokenizers = "^0.19.1"
 llama-index-core = "^0.11.0"
+llama-index-llms-openai = "^0.2.0"
```

Comment:
we need to add the upstage reader as a dependency now right?

Reply:
(should also bump the version of the integration package here)

Reply:
Or, maybe we need to publish the reader first actually

Reply:
Yeah you are right. we should publish reader first.
> we need to add the upstage reader as a dependency now right?

yes, we need to add it